public/js/components/Editor.js

```diff
@@ -152,8 +143,7 @@ module.exports = connect(
       selectedSource: selectedSource,
       selectedSourceOpts: getSelectedSourceOpts(state),
       sourceText: getSourceText(state, selectedId),
-      breakpoints: getBreakpointsForSource(state, selectedId),
-      pause: getPause(state)
+      breakpoints: getBreakpointsForSource(state, selectedId)
```

Comment:
good point, there is. I will fix that in a follow-up PR (that I'm already working on)